Core/scripts/dirac-create-distribution-tarball.py

```diff
@@ -281,6 +281,8 @@ def __checkoutFromGit( self ):
     self.replaceKeywordsWithGit( fDirName )
 
     shutil.rmtree( "%s/.git" % fDirName )
+    shutil.rmtree( "tests" )
```

Comment:
2 things:
* just to point out that it will throw an exception if the directory does not exist
* This will remove the directory 'tests' from the place where you started the command, so maybe an absolute path is safer